media-video/rtmpdump/rtmpdump-9999.ebuild

```diff
@@ -24,6 +24,9 @@ DEPEND="ssl? (
 	)"
 RDEPEND="${DEPEND}"
 
+DOCS=( README ChangeLog )
+HTML_DOCS=( rtmpdump.1.html rtmpgw.8.html )
```

Comment:
Strictly speaking, I'm not sure if there's a point in installing HTML versions of man pages. We already install man pages, don't we?